recipes-ni/ni-boot-time/ni-boot-time.bb

```diff
@@ -0,0 +1,39 @@
+SUMMARY = "Ptest for measuring and logging the boot time"
+SECTION = "tests"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+inherit ptest
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+
+S = "${WORKDIR}"
+
+ALLOW_EMPTY:${PN} = "1"
+
+DEPENDS += "update-rc.d-native"
+RDEPENDS:${PN}-ptest += "bash gawk python3 python3-pip"
+RDEPENDS:${PN}-ptest:append:x64 += "fw-printenv"
+RDEPENDS:${PN}-ptest:append:armv7a += "u-boot-fw-utils"
+
+SRC_URI = "\
+	file://run-ptest \
+	file://upload_results.py \
+	file://zboottime \
+"
+
+do_install_ptest() {
+	install -m 0755 ${S}/run-ptest		${D}${PTEST_PATH}
+	install -m 0755 ${S}/upload_results.py	${D}${PTEST_PATH}
+
+	install -d ${D}${sysconfdir}/init.d
+	install -m 0755 ${S}/zboottime		${D}${sysconfdir}/init.d
+	update-rc.d -r ${D} zboottime		start 99 4 5 .
+}
+
+pkg_postinst_ontarget:${PN}-ptest:append() {
+	python3 -m pip install influxdb
```

Comment:
Do we not have an influxdb python3 package in the feeds? Is that intentional, or have we just not gotten around to making one?

Reply:
We do not and I'm not finding an upstream recipe in the OE layer index either. There is a recipe  for the infulxdb i.e. the database itself but not the python package that interfaces with it.

It's not intentional, just not gotten around to it; arguably it should be added to the backlog.

Reply:
Could you make a TD item for it? Otherwise, I suppose it isn't blocking for this PR.

Reply:
TD #2358572